include/nn/socket.h

```diff
@@ -142,8 +142,8 @@ void nnsocketPoll(pollfd*, ulong, int);
 void nnsocketFcntl(int, int, ...);
 void nnsocketInetPton(int, const char*, void*);
 const char* nnsocketInetNtop(int af, const void* src, char* dst, u32 size);
-s32 nnsocketInetAton(const char* addressStr, InAddr* addressOut);
-char* nnsocketInetNtoa(InAddr);
+s32 nnsocketInetAton(const char* addressStr, in_addr* addressOut);
+char* nnsocketInetNtoa(in_addr);
```

Comment:
These should be re-checked and verified.

Reply:
Discussion in Discord resulted in "has to be `in_addr` for compatibility with < 7.0.0, and no duplicate definitions are allowed for `nn::socket::InAddr`, so it can only be like this"